intelmq/bots/collectors/mail/collector_mail_attach.py

```diff
@@ -28,11 +31,19 @@ class MailAttachCollectorBot(MailCollectorBot):
     mail_user: str = "<user>"
     rate_limit: int = 60
     subject_regex: str = "<subject>"
+    decrypt: bool = False
+    """ Decrypt the attachment with GPG """
+
+    gpg_passphrase: str = ""
+    """ The private key passhrase """
+
+    gpg_home: str = ""
+    """ Change the GPG home directory """
 
     def init(self):
         super().init()
         if self.attach_regex is None:
-            raise InvalidArgument('attach_regex', expected='string')
+            raise InvalidArgument("attach_regex", expected="string")
```

Comment:
I don't see how this and other formatting changes are related to the decryption. It makes the diff harder to read and the blames harder to interpret and use.